<?xml version="1.0"?>
<!-- All setup for MOCK sensors only to be run in simulation, NOT on real boat -->
<launch>
    <!-- Sets speedup, which increases simulation speed -->
    <arg name="speedup" default="1.0" />
    <param name="speedup" type="double" value="$(arg speedup)" />

    <!-- Sets ocean current speed and direction -->
    <arg name="ocean_current_speed" default="0.0" />
    <arg name="ocean_current_direction" default="0.0" />
    <param name="ocean_current_speed" type="double" value="$(arg ocean_current_speed)" />
    <param name="ocean_current_direction" type="double" value="$(arg ocean_current_direction)" />

    <!-- Sets the global wind speed and direction -->
    <arg name="globalWindSpeedKmph" default="1.0" />
    <arg name="globalWindDirectionDegrees" default="45.0" />
    <param name="globalWindSpeedKmph" type="double" value="$(arg globalWindSpeedKmph)" />
    <param name="globalWindDirectionDegrees" type="double" value="$(arg globalWindDirectionDegrees)" />

    <node pkg="local_pathfinding" type="MOCK_sensors.py" name="MOCK_sensors" />
</launch>

